db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/SYSTABLESRowFactory.java

```diff
@@ -257,6 +257,27 @@ public ExecRow makeRow(TupleDescriptor td,
 		/* Build the row to insert  */
 		row = getExecutionFactory().getValueRow(SYSTABLES_COLUMN_COUNT);
 
+		setRowColumns(row, tableID, tableName, tabSType, schemaID, lockGranularity, tableVersion, columnSequence,
+				delimited, escaped, lines, storedAs, location, compression, isPinned, purgeDeletedRows);
+		return row;
+	}
+
+	public static void setRowColumns(ExecRow row,
+									 String tableID,
```

Comment:
are these... tabs? ;-)

Reply:
:) yeah, the whole file uses tabs and my IDE is smart enough to follow whatever whitespace convention the file has, I would leave it as-is for now. Should do a major refactoring to all these files to follow a consistent style.